### DISCUSSION TITLE
Widget background color is not transparent

### DISCUSSION THREAD

**Discussion by NotchApple1703** [TomSchimansky/CustomTkinter#1459] [General]
Is there any way to make the corner of the button widget transparent?

![image](https://user-images.githubusercontent.com/96767159/227493871-36f7f1ad-e66c-4f8b-bc2e-092120ee8904.png)

**Comment by Abdulrhman567** [comment]
have you tried to set the bd=1 and the highlightthickness=1 or 0??

**Comment by Akascape** [comment]
@NotchApple1703 The background portion of the button cannot be transparent because all tkinter canvas widgets are square in shape. bg_color is just to camouflage with the background frame color, but it will not work if your background is an image. 

So change the bg_color of your button according to the image color. You can also specify each corner color by using `background_corner_colors=(color1, color2, color3, color4)`.

**Comment by Akascape** [comment]
@NotchApple1703  A new solution for this issue: https://github.com/TomSchimansky/CustomTkinter/discussions/2214
